src/ctim/schemas/weakness.cljc

```diff
@@ -141,6 +141,11 @@
     (f/entry :effectiveness_notes c/MedString)))
   :reference "[PotentialMitigationsType](https://cwe.mitre.org/documents/schema/#PotentialMitigationsType)")
 
+(def-map-type Note
+  (f/required-entries
+   (f/entry :type v/NoteType)
+   (f/entry :note [c/Markdown])))
```

Comment:
Why an array of `c/Markdown`? You can create paragraph directly with the Makdown format.

Reply:
Yeah it definitely should not be an array of `c/Markdown`.  There's not 1 `:type` and n `:note` fields.  There are n `:type` and m `:note` fields.

Reply:
because we can! No seriously I have no preference myself, just that the array is more premissive in the future vs a single value, I take the votes!